build_errors_schema.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cargo clean && cargo build --manifest-path near/Cargo.toml
```

Comment:
Why do you need `cargo clean`? It's takes more than 10 minutes to rebuild after this?
Also. You should put this script under the `scripts`

Reply:
moved to scripts